PacificPower/readPP.js

```diff
@@ -70,7 +70,30 @@ let continueVarMonthly = 0;
 let page = "";
 let pp_meter_id = "";
 
+let pp_meter_exclude_yes = [];
+let pp_meter_exclude_no = [];
+let pp_meter_exclude_not_found = [];
+
 (async () => {
+  // fetch the meter exclusion list
+  const pp_meter_exclusion_list = await axios({
+    method: "get",
+    url: `${process.env.DASHBOARD_API}/ppexclude`,
+  })
+    .then((res) => {
+      console.log(
+        `RESPONSE: ${res.status}, TEXT: ${
+          res.statusText
+        }, DATA: ${JSON.stringify(res.data)}`,
+      );
+      return res.data;
+    })
+    .catch((err) => {
```

Comment:
In case an error is detected while getting the exclusion list, should the webscraper stop execution entirely, or try to execute like normal but ignore any exclusion logic?

Maybe we can discuss more during next meeting, try to have consistent logic for that between the missing data / exclusions.